### DISCUSSION TITLE
Google Season of Docs 2023 Proposal

### DISCUSSION THREAD

**Discussion by kathatherine** [conda/conda-dot-org#73] [Announcements]
# Writing content for the upcoming conda.org

**We have selected our tech writer for this proposal and have been in contact with them. We sincerely thank all of you for your interest in our project. We had over 200 applications for this project and are overwhelmed and humbled by that. It's unfortunate that we are only able to select one of you. Everyone is welcome to stay involved with this project outside of Google Season of Docs and participate in our community events, if they wish! Thank you.**

**See our announcement [here](https://github.com/conda-incubator/conda-dot-org/discussions/125).**

## About conda

Let's say you're a Python developer. You're updating a project that runs in Python 3.6, but at the same time you're developing something new that uses Python 3.10. It can be really cumbersome to uninstall and reinstall those versions of Python every time you switch between projects, not to mention all of the packages Python needs to run (its dependencies). And speaking of dependencies, many of the code libraries/packages you use have dozens of their own packages that they are dependent on and they all update at different times. It's a mess and it's slowing you way down.

Enter conda. Conda is an open-source package and environment management software that can solve all of those problems for you. The conda tool enables developers to create dozens of different environments (or hundreds, if you really wanted to), each of which can contain different installations of Python, R, IDEs, packages, etc. Conda also has a component called "the solver" that works during environment creation or package install to figure out any dependencies you might be lacking and ask if you want to install them, too.

### How is the conda community organized?

The conda organization maintains the conda package and environment manager and supports conda-adjacent community projects in its [GitHub organization](https://github.com/conda). Conda has an GitHub incubator project for community-led initiatives, one of which is the new conda.org website.

### The problem

The upcoming conda.org website needs more content to serve as a useful resource to members of the conda community. We want conda.org to be a place users can learn more about conda, make announcements, and participate more easily in conda's open-source community.

### Your project's scope

This project will:

- Ensure the front page of the website has clear information and instruction.
- Add to the FAQ (using the GitHub issues that are already created and adding more issues as necessary).
- Research and collate information about all aspects of the conda community. This should include:
    - links to organizational partners, like conda-forge and bioconda, and information about them.
    - community calendar information.
    - community meeting information.
    - communication methods.
- Expand upon our existing "getting started using the conda ecosystem" page that is specifically geared towards newer users.
- Audit the existing conda.io documentation and determine appropriate links to exist in a Learn page.
- Write at least one blog post on a conda-related topic (or perhaps even on the process of researching content for the site).

Work that is out of scope for this project:

- Creating events
- Any website infrastructure work

### How would we measure success?

- conda.org's front page and getting started page presents useful information to first-time visitors that has been approved by the community.
- Most (or all) of the FAQ issues are complete and merged.
- An audit of the existing content in the conda.io documentation is presented and proposes a strategy to populate the Learn page in conda.org.
   - The community gives feedback and approval for the Learn content proposal and it is written accordingly.
- Informative content is present about community partners, calendars, and meetings.
- A new blog post is published.

### Timeline

We assume the tech writer will put in part time hours (10-15 hours/week) during this time.

| Dates             | Action items                                                                |
| ----------------- | --------------------------------------------------------------------------- |
| May               | Technical writer is hired, onboarding                                       |
| June-July         | Research into conda community, user needs for FAQ/support page              |
| August            | Creation/organization of page content                                       |

### Project budget

| Budget item      | Amount    | Running total  |
| ---------------- | --------- | -------------- |
| Technical writer | $6000     | $6000          |
| TOTAL            |           | $6000          |

### Skills needed

Required:

- Proficiency in written English
- Experience with documentation organization or information architecture

Nice to have:

- Knowledge about Python, `conda`, and/or packaging concepts
- Previous contributions to Docusaurus-based websites
- Comfortable with Git, Github and pull request-driven workflows

### Volunteers

- @kathatherine - Main point of contact. Katherine has been technical writing for over ten years and can be a brainstorming partner and copyeditor as needed.

The [conda.org team](https://github.com/orgs/conda-incubator/teams/conda-dot-org/members) has several other regular contributors who will also be able to help with pull request reviews, alongside Katherine.

### Contact info

Technical writers interested in working on this project should send an email to kkinnaman@anaconda.com. Please include links to your technical writing work or portfolio/résumé/CV. **Note: The application cut-off for this proposal is 28 April 2023, 17:00 UTC. We will announce the chosen applicant by 10 May.**

If you have any questions, feel free to ask in our Element room: https://app.element.io/#/room/#conda.org:matrix.org.

**Comment by octonawish-akcodes** [comment]
I am interested How can I proceed.

**Reply by kathatherine** [reply]
Please see the Contact info section of the Proposal above!

**Comment by muffato** [comment]
Nice ! I hope you will remove the confusion between Conda, MiniConda and AnaConda

**Reply by kathatherine** [reply]
We are going to try our best! I agree it can be confusing and that's one of the things we're hoping to solve with an informative conda website.

**Reply by Neilblaze** [reply]
@muffato, @kathatherine +1 for this! I have personally found it to be quite a hassle, so I hope that potential mentees will now have the opportunity to fix it :)

**Comment by tripur3121** [comment]
I want to contribute to Conda, could you direct me toward some resources?

**Reply by kathatherine** [reply]
I assume you mean the conda.org project, yes? We have a [contributing guide](https://github.com/conda-incubator/conda-dot-org/blob/main/CONTRIBUTING.md). However, if you want to apply for the project being proposed above, please see the Contact info section of the proposal.

**Reply by tripur3121** [reply]
@kathatherine Thank you for the help.

**Comment by sushantk1274** [comment]


**Reply by Neilblaze** [reply]
This discussion is for **_Season of Docs_** & **not** for **_Summer of Code_**.

**Comment by cubxxw** [comment]
Hi, everybody, I come from China, is a technology and technical documents of loyal fans. I am happy that conda is close to my future career direction, and I enjoy writing technical documentation
I am engaged in the related studies and work, this is my lot address https://github.com/cubxxw

**Comment by kathatherine** [comment]
Hi, all, I edited the Contact info section of the proposal above to include a cut-off date and time for the applications. Thank you!

**Reply by bupd** [reply]
@kathatherine 
I would like to request your kind attention towards the proposal that I have sent to you via email. I would greatly appreciate it if you could review the proposal and provide me with constructive feedback.

Thank you very much for your time and consideration in this matter.

**Comment by yagyesh-bobde** [comment]
Hello everyone! I am Yagyesh Bobde. I am a web developer but I have some experience in technical writing and I want to take part in season of docs in conda project. Should I directly start with the proposal or is there a getting started guide or resources page that I should follow first?

**Reply by bupd** [reply]
We have a [contributing guide](https://github.com/conda-incubator/conda-dot-org/blob/main/CONTRIBUTING.md). However, if you want to apply for the project being proposed above, please see the Contact info section of the proposal.

**Reply by yagyesh-bobde** [reply]
Okay. Thank you

**Comment by Saintdavidking** [comment]
I am interested in this project, How can I proceed?

**Reply by asmitbm** [reply]
Spam comment deleted!

**Comment by ravi0900** [comment]
I'm bit late but I'm also interested. I have sent the email. 🥲

**Reply by asmitbm** [reply]
The decision has already been made. Read the announcement here: https://github.com/conda-incubator/conda-dot-org/discussions/125

But you can still stick around and contribute to the project :)

**Comment by FarhaKousar1601** [comment]
# I'm interested in this project for GSOD. How can I proceed?

**Reply by Pinak-Datta** [reply]
Hi @FarhaKousar1601 ,
This proposal was for the 2023 Season. 
But you can still stick around, and find ways to contribute to this amazing Repo! 😄

**Comment by Yash8957** [comment]
I want to contribute for this Repository very much interested for this.What are the specific things and skill sets needs to be mentioned in proposal while submission !
